src/components/data.js

```diff
@@ -0,0 +1,42 @@
+import germany from '../img/Germany.png';
+import france from '../img/France.png';
+import Spain from '../img/Spain.png';
+import Italy from '../img/Italy.png';
+import Netherland from '../img/Netherland.png';
+import greatBritain from '../img/greatBritain.png';
+
```

Comment:
- [ ] There is an error raised while running the dev server and that's happened because there is a typo mistake in the name of the image, please fix that.

![image](https://user-images.githubusercontent.com/68796232/225487429-a7738979-0335-4dd2-9894-3f01d8a51979.png)

Reply:
Hey @Omar-Muhamad, would you please check it from the live link again? I'm not sure why I can't see the error 
![image](https://user-images.githubusercontent.com/96848068/225536477-54f47ac3-cb26-41b6-8f7e-00880d750756.png)

Reply:
and this is the importation:
I checked it and I wrote it correctly(the image is in the left side as you can see and the URL is in the right side):
![image](https://user-images.githubusercontent.com/96848068/225539675-9d652231-0369-4fe2-a2f1-e7926e92cbd6.png)

Reply:
Hi @mohisa302, **Omar-Muhamad** was correct, 🙏
I'm sorry I didn't point out that in the previous review, the name of your image is `Greatbritain` not `greatBritain`. please kindly fix that to get your work approved. 😊

- Your import
![image](https://user-images.githubusercontent.com/78251927/225542503-ec225e2c-bcb5-4e99-a9b5-f8fbbfa2cd7f.png)

- Your image folder
![image](https://user-images.githubusercontent.com/78251927/225541592-173e919d-dded-4e09-b559-92a3933eca07.png)

Reply:
Hello @abdulhamiid  thanks for your answer, would you please tell me the exact path of image folder? I want to fix it but I can't find img with the Greatbritain name. I have two image folders:

- one in the src/img :
- 
![image](https://user-images.githubusercontent.com/96848068/225545548-1cc52720-97ea-409f-a3ff-69a0a903810d.png)

- second in the static/media:

![image](https://user-images.githubusercontent.com/96848068/225545740-11016d09-a08b-48f8-85be-49f26ca548a5.png)

I can't find Greabritain name 😞

Reply:
Thanks, guys for all your help 🙏  I'm not sure about the reason but when I pulled it doesn't update. I remove whole the folder and pulled it again and Itgot refreshed 😄